blog/_posts/2017-09-12-Issue-253.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Issue 253
+layout: post
+created_at: Tue September 12 2017
+permalink: blog/2017-09-12-Issue-253
+current: blog
+author: Marie and Ines
+twitter: nk42_
+categories: student-posts
+---
+
+
+Our Project:  [Foodsaving.world](https://foodsaving.world) in the Backend   
+Repository:   <https://github.com/yunity/foodsaving-backend>   
+
+![Retro-Meeting](/img/blog/2017/retro_meeting.jpg)
+_A part of our great team after our first retro meeting, Photo by Daniel Temme_
+
+<br>
+
+## The Plan:
+Before our Rails Girls Summer of Code started, we made a great plan what we will do:
+
+1. create Feedback field to Food Pickup ( [Issue #235](https://github.com/yunity/foodsaving-backend/issues/253) )
+2. make trusted users/group admins    
+3. create statistics for pickups that were done by users in kilograms   
+4. build a quiz framework   
+
+Optional: Android App, shared food & other things
+
+Yay!
+
+<br>
+
+## The Reality
+We found out that we had no clue how difficult such an Issue can be! I mean like ABSOLUTLEY no clue. Coming from 'Codecademy', 'Learn Python the Hard Way' and some small pet projects we thought we would solve the first Issue within 1 week.    
+
+We needed 9 weeks until our first (noteable) Pull request was accepted. So, how was that possible? 
+
+<br>
+
+## The First Issue
+
+Our first task was to create Feedback field to Food Pickups. We spent a lot of our time learning about: Python, Django Framework, Django REST Framework, Swagger, how to troubleshoot our Git merge conflicts and how to solve our Docker complaints.
+
+
+#### We Have Learnt:    
```

Comment:
What we have learned:    
 +1. There are no shortcuts - we have to do Django tutorial and Django REST tutorial first  
 +2. Working both on the same branch at the same time will not speed up our progress! We are going to have many merge conflicts instead, and we have to solve them all. This takes many, many hours :)  
 +3. We know what a server, a browser, a database, models, migrations, HTTP requests, query sets, filters and all these fancy words are; however, we did not know how they are connected before.
 +4. And finally: This Issue is not as small as it looks!